src/Admin/Block/AbstractBlockAdmin.php

```diff
@@ -21,7 +24,7 @@
     /**
      * @var string
      */
-    protected $translationDomain = 'CmfBlockBundle';
+    protected $translationDomain = 'CmfSonataAdminIntegrationBundle';
```

Comment:
technically we could, but i think its not needed. unless we do one catalog per topic, people can just overwrite the CmfSonataAdminIntegrationBundle catalog in app/Resources. i think that is good enough.